src/time/list-columns.vue

```diff
@@ -134,6 +143,18 @@ export default {
     this.scrollToSelected(0);
   },
   methods: {
+    getDaysList() {
+      const la = generateOptions(31, this.dayStep, this.dayOptions).map(num => {
+        const today = new Date();
+        today.setMilliseconds(this.date.getMilliseconds());
+        today.setSeconds(this.date.getSeconds());
+        today.setMinutes(this.date.getMinutes());
+        today.setHours(this.date.getHours());
+        const value = today.setDate(today.getDate() + num);
+        return { value, text: padNumber(num) };
+      });
+      return la;
```

Comment:
@Xanngo I think we may return directly result of `generateOptions`

Reply:
@growson 
O_O
That was debugging code! :see_no_evil: